### PR TITLE
fix: Dependency update (salesforce/acdc#64)

### PR DESCRIPTION
**Pull request by steveblackmon**
resolves salesforce/acdc#64

### CODE REVIEW

File: .sdkmanrc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+java=11.0.20-zulu
+sbt=1.8.2
+scala=2.13.12
```
Comment (steveblackmon) [inline .sdkmanrc:3]: unnecessary but useful

File: build.sbt [hunk 2]
```diff
@@ -53,6 +53,10 @@ lazy val ws = (project in file("acdc-ws")).
       // the transitive jackson dependencies from play framework on has security vulnerabilities
       "com.fasterxml.jackson.core" % "jackson-databind" % "2.15.3",
       "com.fasterxml.jackson.module" %% "jackson-module-scala" % "2.15.3"
-    )
+    ),
+    assemblyMergeStrategy in assembly := {
+      case PathList("META-INF", _*) => MergeStrategy.discard
+      case _                        => MergeStrategy.first
+    }
```
Comment (steveblackmon) [inline build.sbt:60]: fixes assembly errors i encountered building with a clean .ivy cache

File: project/plugins.sbt [hunk 3]
```diff
@@ -2,9 +2,9 @@ addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "1.1.0")
 
 addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.9.1")
 
-addSbtPlugin("com.eed3si9n" % "sbt-buildinfo" % "0.10.0")
+addSbtPlugin("com.eed3si9n" % "sbt-buildinfo" % "0.11.0")
 
-addSbtPlugin("de.heikoseeberger" % "sbt-header" % "5.6.0")
+addSbtPlugin("de.heikoseeberger" % "sbt-header" % "5.10.0")
```
Comment (steveblackmon) [inline project/plugins.sbt:8]: the prior plugin versions are now missing in maven central, so version bumping them as well

**Review by steveblackmon — COMMENTED** [review]
general comments from PR author

**Comment by steveblackmon** [comment]
@realstraw please review (/merge?) at your convenience.  TY

**Review by elhametemad — APPROVED** [review]

**Review by orend — APPROVED** [review]

File: build.sbt (hunk 2 above)
Comment (realstraw) [reply build.sbt:60]: you sure? CircleCI with clean cache doesn't have any issues with assembly

**Pull request closed without merging**